app/Core/Language.php

```diff
@@ -185,20 +185,22 @@ public function readIni(): array
 
         $mainLanguageArray = parse_ini_file(static::DEFAULT_LANG_FOLDER . 'en-US.ini', false, INI_SCANNER_RAW);
 
-        // Complement english with english customization
-        $mainLanguageArray = $this->includeOverrides($mainLanguageArray, static::CUSTOM_LANG_FOLDER . 'en-US.ini');
+        foreach ($languageFiles = self::dispatch_filter('language_files', [
```

Comment:
Does this ensure that it will fall back to english if a translation is not in a translation file?